docs/TrainTest.md

```diff
@@ -37,12 +37,24 @@ In general, both the training and testing include the following steps:
 > CUDA_VISIBLE_DEVICES=0,1,2,3,4,5,6,7 \\\
 > python -m torch.distributed.launch --nproc_per_node=8 --master_port=4321 basicsr/train.py -opt options/train/EDVR/train_EDVR_M_x4_SR_REDS_woTSA.yml --launcher pytorch
 
+or
+
+> PYTHONPATH="./:${PYTHONPATH}" \\\
```

Comment:
As `PYTHONPATH` has already been declared in dist_train.sh, remove it in the cmd